OAuth/ResourceOwner/ZenmoneyResourceOwner.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
+
+use Buzz\Message\RequestInterface as HttpRequestInterface;
+use HWI\Bundle\OAuthBundle\Security\Core\Authentication\Token\OAuthToken;
+use HWI\Bundle\OAuthBundle\Security\OAuthUtils;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * ZenmoneyResourceOwner.
+ *
+ * @author Dmitry Matora <dmitry.matora@gmail.com>
+ */
+class ZenmoneyResourceOwner extends GenericOAuth1ResourceOwner
```

Comment:
`final`

Reply:
That will also not match all the rest files.

Reply:
Well, this is not a big deal.